src/mame/layout/dd9.lay

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0"?>
+<!--
+license:CC0-1.0
+-->
+<mamelayout version="2">
+	<!-- general grey backdrop -->
+	<element name="greyback">
+		<rect>
+			<bounds left="0" top="0" right="1" bottom="1" />
+			<color red="0.1" green="0.1" blue="0.1" />
+		</rect>
+	</element>
+
+	<element name="border">
+		<rect>
+			<bounds left="0" top="0" right="1" bottom="1" />
+			<color red="0.05" green="0.05" blue="0.05" />
+		</rect>
+	</element>
+
+	<element name="black">
+		<rect>
+			<bounds left="0" top="0" right="1" bottom="1" />
+			<color red="0" green="0" blue="0" />
+		</rect>
+	</element>
+
+	<element name="white">
+		<rect>
+			<bounds left="0" top="0" right="1" bottom="1" />
+		</rect>
+	</element>
```

Comment:
There’s no point putting the implicit unit square bounds when you just want to fill the element.